preditor/gui/loggerwindow.py

```diff
@@ -439,6 +490,35 @@ def workbox_for_name(cls, name, show=False, visible=False):
 
         return workbox
 
+    def workbox_for_id(self, workbox_id, show=False, visible=False):
+        """Used to find a workbox for a given id.
+
+        Args:
+            workbox_id(str): The workbox id for which to match when searching
+                for the workbox
+            show (bool, optional): If a workbox is found, call `__show__` on it
+                to ensure that it is initialized and its text is loaded.
+            visible (bool, optional): Make the this workbox visible if found.
+        """
+        # pred = self.instance()
```

Comment:
I've seen a couple of these in this code, are they for a reason or leftover from development?